java/vector/src/test/java/org/apache/arrow/vector/TestOversizedAllocationForValueVector.java

```diff
@@ -112,10 +112,10 @@ public void testVariableVectorReallocation() {
     try {
       vector.allocateNew(expectedAllocationInBytes, 10);
       assertTrue(expectedOffsetSize <= vector.getValueCapacity());
-      assertTrue(expectedAllocationInBytes <= vector.getBuffer().capacity());
+      assertTrue(expectedAllocationInBytes <= vector.getDataBuffer().capacity());
```

Comment:
Is this correct? it is checking only the data buffer capacity i believe

Reply:
Why this is checking DataBuffer and the check after reAlloc is checking OffsetBuffer?

Reply:
I think checking the offset buffer was a typo on my part, this should be checking the data buffer capacity - fixed now

Reply:
Cool. Thanks